frontend/common/src/components/Auth/AuthenticationContainer.tsx

```diff
@@ -49,6 +52,16 @@ const logoutAndRefreshPage = (
   localStorage.removeItem(REFRESH_TOKEN);
   localStorage.removeItem(ID_TOKEN);
 
+  if (postLogoutUri) {
+    if (!postLogoutUri.startsWith("/")) {
+      defaultLogger.warning(
+        `Tried to set an unsafe uri as postLogoutUri: ${postLogoutUri}`,
+      );
+    } else {
+      sessionStorage.setItem(POST_LOGOUT_URI, postLogoutUri);
```

Comment:
Nice!  Storing it in sessionStorage should help use with the two "go to login-info first" issues we put in the backlog yesterday!